tools/validators/instance_validator/validate/telemetry_validator.py

```diff
@@ -36,12 +39,14 @@ class TelemetryValidator(object):
 
   def __init__(self, entities, timeout, callback):
```

Comment:
without args defined in the header comment I don't know what entities is.  Please add docs.

Reply:
done